src/Network/Network/ChangeLog.md

```diff
@@ -22,6 +22,8 @@
 * Added support for new Application Gateway SKU type, Basic SKU
 * Onboarded `Microsoft.EventGrid/partnerNamespaces` to private link cmdlets
 * Onboarded `Microsoft.EventGrid/namespaces` to private link cmdlets
+* Fixed bug in `NewAzureApplicationGatewayFirewallCustomRuleGroupByVariable` to add "GeoLocation" as a valid input for VariableName
+* Added breaking change message for parameter `VariableName` in `NewAzureApplicationGatewayFirewallCustomRuleGroupByVariable` tor remove "Geo" as a valid input.
```

Comment:
Typo "tor".

Also, we are not introducing a breaking change anymore correct?

Reply:
yes, this is only a message indicating that it is happening in Nov, I was asked to add this